packages/platforms/accton/arm64/as4224/src/modules-5_6/arm64-accton-as4224-cpld.c

```diff
@@ -92,7 +92,7 @@ enum cpld_type {
 };
 
 enum as4224_platform_id {
-    AS4224_48X,
+    AS5114_48X,
```

Comment:
Could this be a separate commit with an explanation?

Reply:
Rollback to previous version since this is not related to kernel 5.10.